example/src/index.js

```diff
@@ -1,4 +1,4 @@
-import { LenraApp } from '@lenra/client';
+import { LenraApp, Listener } from '@lenra/client';
```

Comment:
I get this error when running the "npm run build" command :

```console
jonas@macbook-air example % npm run build

> lenra-example@0.0.0 build
> esbuild src/index.js --bundle --outfile=dist/bundle.js

✘ [ERROR] No matching export in "../dist/index.js" for import "Listener"

    src/index.js:1:19:
      1 │ import { LenraApp, Listener } from '@lenra/client';
        ╵                    ~~~~~~~~

1 error
```

Reply:
Have built the lib first ?

Reply:
I've created a good first issue to do it later: https://github.com/lenra-io/client-lib-js/issues/12